nexus/src/nexus.rs

```diff
@@ -2706,9 +2741,11 @@ impl Nexus {
     /// List existing timeseries schema.
     pub async fn timeseries_schema_list(
         &self,
+        opctx: &OpContext,
         pag_params: &TimeseriesSchemaPaginationParams,
         limit: NonZeroU32,
     ) -> Result<dropshot::ResultsPage<TimeseriesSchema>, Error> {
+        opctx.authorize(authz::Action::Read, &authz::FLEET).await?;
```

Comment:
CC @bnaecker -- like with updates, I made this pretty conservative.  You have to have read permissions at the Fleet level to list schema.  We can better define this later.  Let me know if this seems okay for now.

Reply:
That seems totally fine for now, we'll come back to it when we describe how to organize metrics. There's _no concept_ of permissions on individual timeseries now, and I've not thought hard about whether or how to do that.